dynatrace-oneagent/deploy-dynatrace-oneagent.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
 set -e
 
+PLATFORM=$1
+if [ "$PLATFORM" = "KUBERNETES" ]; then
```

Comment:
FYI, we also need to reflect the change for `cr.yaml` file that is downloaded (further down below):
```
curl -o cr.yaml https://raw.githubusercontent.com/Dynatrace/dynatrace-oneagent-operator/master/deploy/cr.yaml
```

Let's also **not** use `master`, but a specific release.